apps/fhir/bluebutton/utils.py

```diff
@@ -174,6 +174,25 @@ def generate_info_headers(request):
     return result
 
 
+def set_header_content_type(headers={}, ct_format=None):
+    """
+    Set the Content-Type to json or xml
+    if not specified we will default to json
+    :param request:
+    :param header:
+    :return: header
+    """
+
+    if 'xml' in ct_format.lower():
```

Comment:
We're only using JSON now, so this clause (and actually the whole function) can be removed. 

Also, this line will cause an error if ct_format is not included or set to None.